app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CategoriesController < ApplicationController
+  def show
+    @category = Category.find(params[:id])
+    @products = @category.products
```

Comment:
You have access to the list of products through the `@category` object so you don't really need to use another instance variable here